### PR TITLE
Generating docs for Akamai v1.0.0

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: content/docs/reference/pkg/akamai/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "akamai"
-title_tag: "Package akamai"
+title: "Akamai"
+title_tag: "Package Akamai"
 meta_desc: "A Pulumi package for creating and managing akamai cloud resources."
```
Comment (davidwrede) [inline content/docs/reference/pkg/akamai/_index.md:4]:
```suggestion
meta_desc: "A Pulumi package for creating and managing Akamai cloud resources."
```

**Review by davidwrede — APPROVED** [review]
Minor nit, but otherwise gtg (from my perspective).

File: content/docs/reference/pkg/akamai/_index.md (hunk 1 above)
Comment (stack72) [reply content/docs/reference/pkg/akamai/_index.md:4]: I need to check the codegen for this - this was created by that

Comment (davidwrede) [reply content/docs/reference/pkg/akamai/_index.md:4]: Yeah, I wasn't sure if it was possible to hand author this, or if the change needs to be made upstream.

**Pull request merged**